lib/components/navigation/SideBar.tsx

```diff
@@ -83,7 +109,34 @@ const SideBar: FC<SideBarProps> = ({ open, onClose }) => {
 
           <div className="flex flex-col gap-3">{generateSiteLinks()}</div>
         </div>
-        <div className="mt-auto flex flex-col gap-3">
+        {isEditMode && (
+          <button
+            onClick={handleEditClick} // TODO: change to edit page navigation
+            disabled={isSelected == ''}
+            className="mt-5 rounded-md border-2 border-amber text-center"
+          >
+            <div className="mx-0 rounded-md px-0 py-3 text-center text-gray-600 transition-all hover:bg-ocf-gray-1000 hover:text-gray-700">
+              <p className="text-center text-base font-medium text-amber">
+                Continue to editing site
+              </p>
+            </div>
+          </button>
```

Comment:
can we use our actual button here? also make it hidden if there isn't any site selected